chalicelib/checks/deployment_checks.py

```diff
@@ -191,3 +193,18 @@ def deploy_application_to_beanstalk(connection, **kwargs):
         cleanup_tempdir(repo_location)
 
     return check
+
+
+def who_is_ff_staging():
```

Comment:
I made [utils PR #91](https://github.com/4dn-dcic/utils/pull/91) so (when that goes in) you can do
```
from dcicutils.beanstalk_utils import compute_ff_stg_env
```
to get this functionality.

Reply:
Pulled this in today.